index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+
+    <link href="https://fonts.cdnfonts.com/css/poppins" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+    <div class="container">
+        <header class="app-bar">
+            <div class="status-bar"><img alt="status bar" src="images/statusbar.png"></div>
+            <nav>
+                <a href="index.html"><h2>Jonas</h2></a>
+                <div>
+                    <a><img alt="overflow-image" src="images/icon_menu_img.png"></a>
+                </div>
+            </nav>
+        </header>
+
+        <section class="header-section">
+            <h1 class="title">I'm Jonas <br>Glad to see you</h1>
+            <p class="bio">I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and 
+                experience! If you like what you see and have a project 
+                you need coded, don’t hestiate to contact me.</p>
+            <p class="social-title">LET'S CONNECT</p>
+            <div class="social-media">
+                <ul>
+                    <li>
+                        <a href="https://twitter.com/jonaswullah" target="_self">
+                            <img src="images/twitter.svg" alt="twitter handle">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="https://linkedin.com/in/jonas-wullah" target="_self">
+                            <img src="images/linkedin.svg" alt="linkedin handle">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" target="_self">
+                            <img src="images/medium.svg" alt="medium handle">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" target="_self">
+                            <img src="images/angellist.svg" alt="angellist handle">
+                        </a>
+                    </li>
+                </ul>
+            </div>
```

Comment:
- [ Optional ] Kindly I recommend you  to add the reaming "GitHub" social icons just like the figma design .

Your Social Icons Look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/206188016-18ed6d14-d6d1-4280-8148-a0817921c15e.PNG)

The Figma Social Icons Look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/206188192-a431e5ef-ac50-45ad-bd83-beee01a084c8.PNG)